nengo/spa/tests/test_module.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+import nengo
+from nengo import spa
+from nengo.spa.utils import similarity
+
+
+class SpaCommunicationChannel(spa.Module):
```

Comment:
I think it's also pretty self explanatory: It is a SPA communication channel module. *duh*